shaolinmkz/iReporter
# #162190238 Build out edit specific redflag record location end-point

#### What does this PR do?
Have the update specific red-flag record location API functional

#### Description of Task to be completed?
Having the endpoint POST  `host:port/api/v1/red-flags/<red-flag-id>/location` working

#### How should this be manually tested?
After cloning the repo, cd into it then:
* RUN `npm test` to see test result
* RUN `npm start` to start the server
* Using Postman test the endpoint POST  `host:port/api/v1/red-flags/<red-flag-id>/location` with the header:
_key:_ Content-Type value: application/json

#### What are the relevant pivotal tracker stories?
[#162190238](https://www.pivotaltracker.com/story/show/162190238)
